restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -32,17 +36,37 @@
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 import java.util.function.UnaryOperator;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 public class DocumentDBTest {
+  private static final Schema schema = buildSchema();
+
   private DocumentDB documentDB;
   private TestDataStore ds;
   private static final ObjectMapper mapper = new ObjectMapper();
 
-  @Before
+  private static Schema buildSchema() {
+    SchemaBuilder__5 schemaBuilder =
```

Comment:
This is an unfortunate side effect of the way [duzzt](https://github.com/misberner/duzzt) generates code. We use it to generate the schema builder. Normally these class names do not surface, but here we use it in a loop, as I wanted to avoid copying the same line 64 times :)